modules/age.nix

```
@@ -187,6 +187,13 @@ in {
         The age executable to use.
       '';
     };
+    pluginPackages = mkOption {
```

I was thinking perhaps it would be nicer to name this `path`, similar to `systemd.services.<name>.path`. On the other hand, the name `pluginPackages` communicates more intent and I believe is more discoverable. Thoughts? :)

---

I agree `plugins` should be used in this case.

```suggestion
    plugins = mkOption {
```